okcupyd/filter.py

```diff
@@ -19,25 +18,26 @@ class Filters(object):
     def __init__(self):
         self.builders = []
         self.keys = set()
-        self._key_to_type = {}
-        self._key_to_values = {}
         self._key_to_string = {}
+        self._key_to_values = {}
+        self._key_to_type = {}
 
     @util.cached_property
     def filter_meta(filters_instance):
         class FilterMeta(util.decorate_all(staticmethod)):
 
-            acceptable_values = None
-            keys = ()
-            types = None
-            descriptions = None
             output_key = None
+            keys = () # must be a container
```

Comment:
So something like

    #: keys must be a container

and Sphinx will add that comment to the documentation for the containing function? I'm guessing it doesn't extract any other information from the context other than what function/class/method the comment is embedded in?